src/test/ReactTestUtils.js

```diff
@@ -178,9 +178,6 @@ var ReactTestUtils = {
    * @return {array} an array of all the matches.
    */
   scryRenderedDOMComponentsWithClass: function(root, classNames) {
-    if (!Array.isArray(classNames)) {
```

Comment:
Thanks @ipeters90 for the detailed explanation.

>if someone makes a mistake and entered the className foo as the only argument

I guess then there should have been a check for `root` as well.